app/controllers/api/v1/patients_controller.rb

```diff
@@ -10,11 +10,26 @@ def sync_from_user
     render json: response, status: :ok
   end
 
+  def sync_to_user
+    patients_to_sync = Patient.updated_on_server_since(latest_record_timestamp, number_of_records)
+
+    next_latest_record_timestamp =
```

Comment:
This variable name is confusing. How about `most_recent_record_timestamp`?

Reply:
Yup, nice catch. This seemed so correct to @govindkrjoshi and me at the time.